src/Compilers/CSharp/Portable/Symbols/Source/SourceLocalSymbol.cs

```diff
@@ -67,6 +68,11 @@ internal class SourceLocalSymbol : LocalSymbol
             _locations = ImmutableArray.Create<Location>(identifierToken.GetLocation());
         }
 
+        internal Binder Binder
+        {
+            get { return binder; }
```

Comment:
This is consistent with other properties in this file and I see no benefit in changing the style.